packages/libproj/meta.yaml

```
@@ -0,0 +1,45 @@
+package:
+  name: libproj
+  version: 8.2.1
+
+source:
+  sha256: 76ed3d0c3a348a6693dfae535e5658bbfd47f71cb7ff7eb96d9f12f7e068b1cf
+  url: https://download.osgeo.org/proj/proj-8.2.1.tar.gz
+
+requirements:
+  run:
+    - sqlite3
```

I think you need to add libtiff here:

```suggestion
    - sqlite3
    - libtiff
```

---

Thanks, that might fix libtiff, but I still need to figure out why the pre-built sqlite3 is failing when the build tries to use it.

---

> Thanks, that might fix libtiff, but I still need to figure out why the pre-built sqlite3 is failing when the build tries to use it.

https://stackoverflow.com/questions/14082152/sqlite3-no-such-file-or-directory

Hmm it seems like pre-built sqlite3 binary doesn't work for 64bit systems. I'm not sure why they don't provide x86-64 binaries. I think we will have to build sqlite3 for host os or add sqlite3 binary in our docker image as you mentioned before...

---

Including a `sqlite3` binary like you would a compiler (using a distro package) seems like the easiest path here...

---

Alright, let's add sqlite3.

---

Can I just add it to the `Dockerfile`? Or does it need a separate PR?

---

I'll open a PR. One of core devs need to trigger the Docker deploy workflow.